cy-drt-dockfile

```diff
@@ -1,5 +1,5 @@
 # use Cypress provided image with all dependencies included
-FROM cypress/base:18.12.1
```

Comment:
is specific version better than latest ? as it may fail randomly with latest . 
 
I see advantage of latest version to keep it update all the time but may have uncertainty irrespective of our changes?

Reply:
Yes good point. I had meant to tie it to actual latest version. I'll do that now